ui/src/app/utils/runtime-utils.tsx

```diff
@@ -263,12 +269,8 @@ const compareDataprocNumberOfWorkers = (oldRuntime: RuntimeConfig, newRuntime: R
 };
 
 const compareAutopauseThreshold = (oldRuntime: RuntimeConfig, newRuntime: RuntimeConfig): RuntimeDiff => {
-  const oldAutopauseThreshold = oldRuntime.autopauseThreshold;
-  const newAutopauseThreshold = newRuntime.autopauseThreshold;
-
-  if (!oldAutopauseThreshold || !newAutopauseThreshold) {
-    return null;
-  }
+  const oldAutopauseThreshold = oldRuntime.autopauseThreshold == null ? DEFAULT_AUTOPAUSE_THRESHOLD_MINUTES : oldRuntime.autopauseThreshold;
```

Comment:
`== null` can have surprising results, would use === when comparing to null

Reply:
yeah.. I was cheating a bit by relying on the fact that `null == undefined`  but it's probably better to explicitly declare both.